### PR TITLE
Add debug local updates

### PR DESCRIPTION
**Pull request by AndrewBoyarsky**

### CODE REVIEW

**Comment by AndrewBoyarsky** [comment]
> rename update-attachment.json to update-attachment.json.example or something like that

done

File: apl-updater/src/main/java/com/apollocurrency/aplwallet/apl/updater/UpdaterUtil.java [hunk 1]
```diff
@@ -18,61 +22,98 @@
...
         return certificates;
     }
 
     public static Certificate readCertificate(String certificateFileName) throws CertificateException, IOException, URISyntaxException {
-        Path certificateFilePath = loadResourcePath(certificateFileName);
         CertificateFactory cf = CertificateFactory.getInstance("X.509");
-        return cf.generateCertificate(Files.newInputStream(certificateFilePath));
+
+        URL resource = getResource(certificateFileName);
+        System.out.println(resource);
```
[the remark this answers is not in the document]
Comment (AndrewBoyarsky) [reply apl-updater/src/main/java/com/apollocurrency/aplwallet/apl/updater/UpdaterUtil.java]: done

File: apl-updater/src/main/java/com/apollocurrency/aplwallet/apl/updater/SimpleUrlExtractor.java [hunk 2]
```diff
@@ -40,8 +40,8 @@ public SimpleUrlExtractor(DoubleDecryptor decryptor) {
 
     @Override
     public String extract(byte[] encryptedUrlBytes, Pattern urlPattern) {
-        Set<UpdaterUtil.CertificatePair> certPairs = certificatePairs != null ? certificatePairs : certificatePairsProvider.getPairs();
-        for (UpdaterUtil.CertificatePair pair : certPairs) {
+        Set<CertificatePair> certPairs = certificatePairs != null ? certificatePairs : certificatePairsProvider.getPairs();
+        for (CertificatePair pair : certPairs) {
```
Comment (leonidstr) [inline apl-updater/src/main/java/com/apollocurrency/aplwallet/apl/updater/SimpleUrlExtractor.java]: NPE if certPairs is null;

Comment (AndrewBoyarsky) [reply apl-updater/src/main/java/com/apollocurrency/aplwallet/apl/updater/SimpleUrlExtractor.java]: done

**Pull request merged**